packages/back-end/src/init/index.ts

```diff
@@ -2,11 +2,13 @@ import mongoInit from "./mongo";
 import licenseInit from "./license";
 import { queueInit } from "./queue";
 import { logger } from "../util/logger";
+import { initializeEventEmitters } from "./event-emitters";
 
 let initPromise: Promise<void>;
 export async function init() {
   if (!initPromise) {
     initPromise = (async () => {
+      initializeEventEmitters();
```

Comment:
I think this might have to go after `mongoInit`.  The event handlers will likely import Model files, which themselves import mongoose.  I'm not 100% sure, but that might mess up the mongoose configuration if that happens before the init call.